src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -843,5 +860,19 @@ internal bool TryGetContainingEditableSpan(SnapshotPoint point, out SnapshotSpan
 
             return false;
         }
+
+        internal TestAccessor GetTestAccessor()
+            => new TestAccessor(this);
+
+        public struct TestAccessor
+        {
+            private readonly InlineRenameSession _inlineRenameSession;
+
+            public TestAccessor(InlineRenameSession inlineRenameSession)
+                => _inlineRenameSession = inlineRenameSession;
+
+            public bool CommitWorker(bool previewChanges)
+                => _inlineRenameSession.CommitWorker(previewChanges);
+        }
```

Comment:
What is this all buying us here? It seems like we went from a nice public Commit method to something that's internal and now we have a TestAccessor for it?